packages/postgres-database/src/simcore_postgres_database/utils_user_preferences.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any
+
+import sqlalchemy as sa
+from aiopg.sa.connection import SAConnection
+from sqlalchemy.dialects.postgresql import insert as pg_insert
+
+from .models.user_preferences import (
+    user_preferences_frontend,
+    user_preferences_user_service,
+)
+
+
+class CouldNotCreateOrUpdateUserPreferenceError(Exception):
+    ...
+
+
+class BasePreferencesRepo:
+    model: sa.Table
+
+    @classmethod
+    async def save_frontend_preference_payload(
```

Comment:
If this is used for both front/user-service preference, I would simply call them `save` and `load` resp.